src/qt/rpcconsole.cpp

```diff
@@ -836,3 +859,7 @@ void RPCConsole::showOrHideBanTableIfRequired()
     ui->banHeading->setVisible(visible);
 }
 
+void RPCConsole::on_toggleNetworkActiveButton_clicked()
```

Comment:
this is not connected to any button, isn't it?

Reply:
yeah, not connected. Good, cleaning